test_plans/optional_kernel_features.asciidoc

```diff
@@ -118,3 +124,21 @@ The first kernel is submitted unconditionally to the device.
 Each of the remaining kernels is submitted to the device only if it supports the kernel's feature / work-group size / sub-group size.
 
 Check that no exception is thrown.
+
+=== Call for work_group_size_hint
```

Comment:
Neither the `work_group_size_hint` nor the `vec_type_hint` attributes are "optional kernel features" as defined in section 5.7.  Why add them to this test plan?

Instead, it seems like they should either be in their own test plan or part of a general test of the SYCL attributes.

Reply:
You are right, I got it mixed up because attributes are heavily used in optional kernel features tests. Moved these new test cases to different test plan "kernel_attributes".